Gemfile

```diff
@@ -60,4 +60,7 @@ group :development, :test do
 
   # Use factory_girl in place of fixtures for generating test objects
   gem "factory_girl_rails", "~> 4.0"
-end
\ No newline at end of file
+end
+
+# Use Simple Form for easier form creation
+gem "simple_form"
```

Comment:
http://www3.pictures.zimbio.com/mp/mA5SbP1W53Ex.gif. Thanks for the info. I'll stop deleting these.